src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -206,6 +206,16 @@ static Config::ModelType getModelType(const std::shared_ptr<const Model>& model)
     return Config::ModelType::Unknown;
 }
 
+void Plugin::apply_rt_info(const std::shared_ptr<const ov::Model>& model, ov::intel_cpu::Config& config) const {
```

Comment:
Logically it should be part of Config API:
Config::apply_model_rt_info(const std::shared_ptr<const ov::Model>& model)

Reply:
Fix, thx